### ISSUE TITLE
 some questions about Z

### ISSUE THREAD

**Issue by Esther-PAN** [hyunjunChhoi/Balanced_Energy#3]
Hello, Thanks for your work！
I have some questions regarding the determination of auxiliary data as majority or minority classes. First, auxiliary data and ID are not overlap, and theoretically unknown (in this case, labels are not provided). 
In the paper, it states, "Z is the weighted sum of the softmax output of the classification model for a sample (i.e., the posterior probability of a class for a given sample), where the weight is the prior probability for the class." 
Could you please explain how the prior probability for Z in the text is obtained? Is "the sum of the softmax output of the classification model for a sample" referring to the probabilities of its several categories in the ID?

**Comment by hyunjunChhoi** [comment]
We can get the prior distribution from inference on auxiliary training set.
Our approach follows fine-tuning strategy as in energy fine-tuning method
We can infer on auxiliary training dataset based on the pretrained model.

Thanks

2024년 4월 19일 (금) 오후 9:22, dengye Pan ***@***.***>님이 작성:

**Comment by Esther-PAN** [comment]
Thank you for your prompt reply~ I'm still a bit confused. 
For image classification tasks, pretrained model here is a classifier trained with id data like CIFAR-10. Am I understanding this correctly?
And what's the significance of the classifications obtained by inputting auxiliary data into the model for inference, since the categories of the auxiliary data are not included in the few labels that the model learns?

**Comment by Esther-PAN** [comment]
For example, the pretrained model is trained using CIFAR-10, where it learns categories from 0 to 9. However, the auxiliary dataset is 80 Million Tiny Image, which includes categories such as cars, flowers, and others.

**Comment by hyunjunChhoi** [comment]
Yes

We use the inference result

Of Auxiliary data into cifar 10 class

We measure the how original network

Confuse Auxiliary data with original

10 class.  Z measure this confusion of Auxiliary data with original
indistribution classes .  We measure Cardinality and constitute prior
distribution

Please refer to the paper in detail.





2024년 4월 20일 (토) 오후 9:17, dengye Pan ***@***.***>님이 작성:

**Comment by Esther-PAN** [comment]
thanks
